content/en/docs/reference/config-api/apiserver-encryption.v1.md

```diff
@@ -94,7 +94,13 @@ Each key has to be 32 bytes long for AES-CBC and 16, 24 or 32 bytes for AES-GCM.
 <thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
 <tbody>
     
-  
+<tr><td><code>apiVersion</code><br/>
```

Comment:
This file is auto-generated. Please don't modify manually.

Reply:
Thanks! I will remove it. Would the change get automatically added at some point? What triggers the auto-gen?

Reply:
New version is generated when a new k8s release is out, maybe a few days post the release.